packages/playwright-core/src/server/firefox/ffPage.ts

```diff
@@ -399,6 +399,10 @@ export class FFPage implements PageDelegate {
     return success;
   }
 
+  async forceGarbageCollection(): Promise<void> {
+    // Not supported in Firefox.
```

Comment:
We don't do that, should be supported to make its way into Playwright.

Reply:
Understood, I'll look into it further.